ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/ReferenceExpr.cs

```diff
@@ -82,6 +82,7 @@ public FSharpSymbolReference Reference
       FSharpImplUtil.SetName(this, name);
 
     ITypeArgumentList ITypeArgumentOwner.TypeArgumentList => TypeArgumentList;
+    private string TypeArgumentsText => TypeArgumentList?.GetText() ?? "";
```

Comment:
Please construct the text properly, like it's done in QualifiedName; we don't want new lines, comments and other stuff in places we'd use it.

Reply:
We agreed to show the "Simplify lambda" tooltip for the case of "complex" qualified names